job-server/src/main/scala/spark/jobserver/AkkaClusterSupervisorActor.scala

```diff
@@ -449,13 +449,13 @@ class AkkaClusterSupervisorActor(daoActor: ActorRef, dataManagerActor: ActorRef,
       contextConfig, Some(resultActor), dataManagerActor))(Timeout(timeoutSecs.second)).onComplete {
       case Failure(e: Exception) =>
         logger.info("Failed to send initialize message to context " + ref, e)
-        cluster.down(ref.path.address)
+        context.stop(ref)
```

Comment:
I am not sure what the exact reasons are, but somehow the behavior between akka-2.4 and akka-2.5 changed. `ref.path.address` contains not the address of the actual actor. In the actor supervisor spec,  the address was `akka://test` shutting the complete cluster down instead of only a single actor.